packages/config/config/devices/0x001d/dzpd3.json

```diff
@@ -74,7 +90,6 @@
 		},
 		"7": {
 			"label": "Locator LED Status",
```

Comment:
Change this to LED Indicator. And add the options. Status mode is LED on when switch is on. Locator mode is LED on when switch if off. Off is Always Off.